ipn/localapi/localapi.go

```diff
@@ -350,6 +352,23 @@ func (h *Handler) serveCheckIPForwarding(w http.ResponseWriter, r *http.Request)
 	})
 }
 
+func (h *Handler) serveCheckRPFilter(w http.ResponseWriter, r *http.Request) {
+	if !h.PermitRead {
+		http.Error(w, "reverse path filtering check access denied", http.StatusForbidden)
+		return
+	}
+	var warning string
+	if err := h.b.CheckRPFilter(); err != nil {
+		warning = err.Error()
+	}
+	w.Header().Set("Content-Type", "application/json")
+	json.NewEncoder(w).Encode(struct {
```

Comment:
I'm not a fan of this unnamed type on both the server and client.

It's a protocol type; it's pretty important.